### DISCUSSION TITLE
quarto.path in VS Code not used?

### DISCUSSION THREAD

**Discussion by astrowonk** [quarto-dev/quarto-cli#8195] [Q&A]
### Description

I expanded the RHEL tarball for Quarto 1.4 but my attempt to tell VS Code extension where my quarto binary is doesn't seem to do anything? I have  `"quarto.path"` set in my remote settings but when I do "quarto preview" or any command for quarto in the VS Code command palette, it just sends a "quarto" command to the Terminal which finds nothing.

I can add the path to my UNIX path to work around this, but what's the point of the `quarto.path` VS code setting if it's not used? Am I doing something wrong?

**Comment by mcanouil** [comment]
Issues regarding the Quarto VSCode extension are to be open in <https://github.com/quarto-dev/quarto/issues>.

Note that without any reproducible example or steps to reproduce, it is very hard to figure what's wrong in your setup.


Your issue might be related to https://github.com/quarto-dev/quarto/issues/213

**Reply by gadenbuie** [reply]
OP's issue was opened at https://github.com/quarto-dev/quarto/issues/352. It's not related to quarto-dev/quarto#213. I added notes here https://github.com/quarto-dev/quarto/issues/352#issuecomment-2120967694.
